Assets/Vimeo/Scripts/Player/VimeoPlayer.cs

```diff
@@ -391,7 +393,30 @@ private void VideoMetadataLoad(string response)
                 Debug.LogError("Video could not be found");
             }
         }
-        
+
+        public IEnumerator Unfurl(string url)
+        {
+            using (UnityWebRequest www = UnityWebRequest.Get(url)) {
```

Comment:
Can we DRY up this request logic with vimeo API somehow? Not a big deal, we can leave it, just starts to get a little messy.

Reply:
https://github.com/vimeo/vimeo-unity-sdk/pull/77/commits/5cdbce2703b4ed0cabb000783d402229e7999e50